build_modules/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # 0.2.2
 
-- Add new `strategy` option for `ModuleBuilder`. If `coarse` is provided a
-  minimum number of modules will be created. Otherwise, a module will be
-  created for each strongly connected component. Note that `coarse` is
-  the default strategy.
+- Add new `ModuleBuilder` strategies. By default the `coarse` strategy is used
+  for all non-root packages and will create a minimum number of modules. This
+  strategy can not be overridden. However, for the root package, the `fine`
+  strategy will be used which creates a module for each strongly
+  connected component. You can override this behavior by providing `coarse`
+  to the `strategy` option.
```

Comment:
nit: add an example of overriding this:

```yaml
targets:
  $default:
    builders:
      build_modules|modules:
        options:
          strategy: coarse
```

Note that we should actually rename the builder in the build.yaml in this package to `build_modules`, so that you don't have to do `build_modules|modules` above.

Reply:
Done.